nsxt/resource_nsxt_policy_ipsec_vpn_local_endpoint.go

```diff
@@ -21,6 +22,45 @@ import (
 	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
 )
 
+var IPSecVpnLocalEndpointSchema = map[string]*metadata.ExtendedSchema{
```

Comment:
Why not follow the common pattern where we use a common func that creates the schema, with flags to add/delete the specifics - e.g `getPolicyCommonSegmentSchema`?